handbook/engineering/code-insights/index.md

```diff
@@ -0,0 +1,34 @@
+# Code insights team (coming soon)
+
+The code insights team will be responsible for building and delivering [**code insights**](https://docs.google.com/document/d/1EHzor6I1GhVVIpl70mH-c10b1tNEl_p1xRMJ9qHQfoc/edit) to engineering leaders, empowering data-driven decisions in engineering organizations.
+
+<img src="./screenshot.svg" alt="Screenshot of a code insights dashboard with graphs" />
+
+## What is code insights?
+
+Code insights is the first feature in Sourcegraph that can tell you things about your code at a **high level**. Here's a [demo](https://www.youtube.com/watch?v=XqeRb6Mc4Co) of a code insights prototype.
+
+Sourcegraph is in the unique position to give these insights because we have universal code search: To know _anything_ about your code at a high level means you must know _everything_ about your code at a low level.
+
+Code insights connects a lot of features that Sourcegraph already has and builds on top of them.
+We go beyond single-step code intelligence and search to connect the full cycle of analyzing (code intelligence), monitoring (code insights), and actionably changing a codebase (campaigns).
+
+Code insights will also allow to combine these Sourcegraph-created metrics with other third-party data sources.
+
+Code insights is the first feature primarily built for non-search-based user personas.
+
+## Goals
```

Comment:
I am hesitant to label these as goals because @christinaforney and I are working hard to align the team on what "goals" mean and this does not fit that definition.

Here are examples of what we want "goals" to be:
- https://about.sourcegraph.com/handbook/engineering/cloud/goals
- https://about.sourcegraph.com/handbook/engineering/code-intelligence/goals

Reply:
This section was taken directly from the web team's Goal page: https://about.sourcegraph.com/handbook/engineering/web/goals#medium-term

Reply:
I've already spoken with @Joelkw and the team about goals, he was more ahead and pushed a nice update to goals right before we started to refine our format. He will be coming back to this to align on format with the team.

Separately, I would only move goals here that are specific to code insights and explicitly remove them from the web team goals to avoid duplication. Perhaps an interim solution is to just link back to the web team goals?

Reply:
```suggestion
## Vision
```

Can we re-brand this as the "vision" for the team?